inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -397,50 +398,71 @@ def _curators_note2marc(self, key, value):
 
 
 @hepnames.over('experiments', '^693..')
-@utils.for_each_value
-@utils.filter_values
-def experiments(self, key, value):
+def experiments(self, key, values):
     """Information about experiments.
 
     Currently the ``id`` subfield stores the name of the experiment. It should
     be moved to storing id of the experiment as soon as experiments are
     available as records.
     """
-    try:
-        start_year = int(value.get('s'))
-    except (TypeError, ValueError):
-        start_year = None
-    try:
-        end_year = int(value.get('d'))
-    except (TypeError, ValueError):
-        end_year = None
-
-    return {
-        'id': value.get('i'),
-        'name': value.get('e'),
-        'start_year': start_year,
-        'end_year': end_year,
-        'status': value.get('z')
-    }
+    # FIXME: change to use the flatten decorator once the new dojson version is
+    # out
+
+    def _get_json_experiments(marc_dict):
+        try:
+            start_year = int(marc_dict.get('s'))
+        except (TypeError, ValueError):
+            start_year = None
+        try:
+            end_year = int(marc_dict.get('d'))
+        except (TypeError, ValueError):
+            end_year = None
+
+        for name in force_force_list(marc_dict.get('e')):
+            yield {
+                'id': marc_dict.get('i'),
+                'name': name,
+                'start_year': start_year,
+                'end_year': end_year,
+                'status': marc_dict.get('z')
+            }
```

Comment:
This doesn't fit in the schema that you should be generating: https://github.com/inspirehep/inspire-next/blob/90032ba4825c6c4dda78c9c2b1d3edf18692b9dc/inspirehep/modules/records/jsonschemas/records/authors.json#L65-L86